src/main/resources/react/src/components/header/nav_main/styles.js

```diff
@@ -0,0 +1,169 @@
+import styled from 'styled-components';
+import logo from 'assets/img/logo.png';
+import { Link } from 'react-router-dom';
```

Comment:
sort by length
for example
import { Link } from 'react-router-dom';
import styled from 'styled-components';

import logo from 'assets/img/logo.png';